### PR TITLE
chore(Body): do not inject styles in Fragment component

### PR DESCRIPTION
**Pull request by MounirDhahri**
### Description

This PR comes to address a potential prop leak introduced in the previous PR https://github.com/artsy/palette-mobile/pull/232

### CODE REVIEW

File: src/elements/Screen/Body.tsx [hunk 1]
```diff
@@ -16,15 +15,17 @@ export const Body: React.FC<BodyProps> = ({
...
+    <Flex flex={1} {...flexProps}>
+      <Flex flex={1} px={fullwidth ? undefined : SCREEN_HORIZONTAL_PADDING}>
+        {scroll ? <ScrollView>{children}</ScrollView> : children}
       </Flex>
-    </Wrapper>
+    </Flex>
   )
+
+  if (disableKeyboardAvoidance) {
+    return Content
```
Comment (MounirDhahri) [inline src/elements/Screen/Body.tsx:27]: I hate conditional wrappers 😄 there is not beautiful way to write them that I know of

**Review by gkartalis — APPROVED** [review]
thanks

File: src/elements/Screen/Body.tsx (hunk 1 above)
Comment (gkartalis) [reply src/elements/Screen/Body.tsx:27]: hmm actually what do you think about something like this:

Comment (gkartalis) [reply src/elements/Screen/Body.tsx:27]:
```
<Wrapper
        {...(disableKeyboardAvoidance
          ? {}
          : { style: { flex: 1 }, behavior: Platform.OS === "ios" ? "padding" : "height" })}
      >
```

With this way we are getting away without conditional wrappers

Comment (MounirDhahri) [reply src/elements/Screen/Body.tsx:27]: not the sexiest but would also work - will update

**Comment by MounirDhahri** [comment]
thanks to you 🌷

**Comment by artsyit** [comment]
:rocket: PR was released in [`v13.2.11`](https://github.com/artsy/palette-mobile/releases/tag/v13.2.11) :rocket:

**Pull request merged**